src/pages/FeelGoodPage.js

```diff
@@ -13,16 +17,29 @@ export default function FeelGoodPage() {
           width="60"
         />
       </GoBackButton>
-      <Header>Scared?</Header>
-      <Text>
-        Don't worry!
-        <br />
-        The monsters under your bed are not that bad...
-        <br />
-        You should be more worried about the monsters in your closet!
-      </Text>
+      <Header>Scared? Not anymore!</Header>
+      <CatContainer>
+        <img
+          src={catUrl?.url}
+          alt="Press the button below to see cute cats!"
```

Comment:
I am not sure whether this is a good alt text, because the img itself shows random cats, right?